runtime/vm/FFITypeHelpers.cpp

```diff
@@ -153,7 +153,12 @@ FFITypeHelpers::freeStructFFIType(ffi_type *ffiType)
 {
 	if (NULL != ffiType) {
 		PORT_ACCESS_FROM_JAVAVM(_vm);
-		if (FFI_TYPE_STRUCT == ffiType->type) {
+		if (FFI_TYPE_STRUCT == ffiType->type
```

Comment:
There is no need to modify the code in `FFITypeHelpers.cpp` given everything with `FFITypeHelpers.hpp/cpp` (will be removed later) are replaced by `LayoutFFITypeHelpers.hpp/cpp` in the FFI specific implementation.

Reply:
Removed changes from FFITypeHelpers.cpp in https://github.com/eclipse-openj9/openj9/pull/19645/commits/1e752555416853814f4beb0cf0eea5f3fe8580bf